techniques/android/MASTG-TECH-0117.md

```diff
@@ -3,43 +3,72 @@ title: Obtaining Information from the AndroidManifest
 platform: android 
 ---
 
-Multiple ways exist to view the contents of the AndroidManifest:
+The [AndroidManifest.xml](../../Document/0x05a-Platform-Overview.md) file is a critical component of any Android application, providing essential information about the app's structure, permissions, components, and configurations. During a security assessment, analyzing the manifest can reveal potential vulnerabilities or misconfigurations that could be exploited by attackers.
+
+The AndroidManifest can be obtained from the APK file, and if you try to open it directly, you might see it's in a binary format. To properly analyze the manifest, you need to extract and decode it into a human-readable XML format.
+
+Different tools extract the manifest in various formats, with some preserving more raw structure while others interpret or modify it during decoding.
+
+## Using @MASTG-TOOL-0018
+
+Use jadx CLI with `--no-src` to extract only resources without decompiling all sources:
+
+```sh
+jadx --no-src -d out_dir MASTG-DEMO-0001.apk
+```
+
+jadx outputs the manifest in full to `out_dir/resources/AndroidManifest.xml`, including the `<uses-sdk>` element which is not included when using other tools like apktool.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" ...>
+    <uses-sdk
+        android:minSdkVersion="29"
+        android:targetSdkVersion="35" />
+```
 
 ## Using @MASTG-TOOL-0011
 
-The full AndroidManifest can be extracted using @MASTG-TOOL-0011:
+The full AndroidManifest can be extracted using apktool:
```

Comment:
But further down you say it's not the full manifest.

Reply:
```suggestion
The AndroidManifest can be extracted using apktool:
```